lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -9,6 +9,7 @@ statsd
 azure-storage==0.32.0
 python-irodsclient==1.1.3
 python-ldap==3.4.0
+ldap3==2.9.1
```

Comment:
Wondering if it makes sense to have both in this file?

Reply:
It doesn't hurt, but to trigger auto installation a `check_ldap3()` method similar to `check_python_ldap()` needs to be added to `lib/galaxy/dependencies/__init__.py` .

Reply:
Cool. Not aware of `check_python_ldap()` method. I can add a method as suggested.